packages/deploy/src/index.ts

```diff
@@ -84,7 +84,7 @@ function writeState(config: DeployConfig, nextState: {}): Promise<void> {
 // ==================== Spec ===================
 
 // Given a path to a project spec, read and validate it.
-async function getSpec(path: string) {
+export async function getSpec(path: string): Promise<any> {
```

Comment:
We should be able to do better than `Promise<any>` here. Are you sure you need to declare the type at all? I'd expet Typescript to be able to infer it.

If you really need an explicit type, you'll have to:
* in `validator.ts`, pull the return type of `parseAndValidate` out into its own interface - maybe called `ValidatedSpec` and export it
* Then here, you can do `Promise<ValidatedSpec>`

Reply:
I've removed the `Promise<any>` - the reason we needed it (to my surprise) was that `Error` type in the validator wasn't exported.